packages/trpc/server/routers/viewer/organizations/listMembers.handler.ts

```diff
@@ -81,9 +82,7 @@ export const listMembersHandler = async ({ ctx, input }: GetOptions) => {
               attribute: {
                 id: filter.id,
               },
-              value: {
-                in: filter.value,
-              },
+              ...makeWhereClause("value", filter.value),
```

Comment:
Filtering by text shouldn't be case sensitive

https://github.com/user-attachments/assets/9f833e55-d394-4e4f-8c90-385bef236923